### ISSUE TITLE
Submit data to vets-api

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/va-forms-system-core#553]
## Description
VAFSC form should properly submit data to vets-api.

## Considerations
- Can be done in tandem with #542, or should be a fast follow after #542 is completed. 
- Work will _likely_ take place largely in vets-website, but some work may need to be done in VAFSC
- Could be focused on cleanup work after ticket #542
- Serialization of data for submission to vets-api likely similar to save in progress (work on save in progress later)

## Tasks
- [ ] Data submits successfully from the burials poc in vets-website to vets-api in a local env

## Acceptance Criteria
- [ ] Tasks have been complete
- [ ] Any PRs have been merged into main.
